src/py/reactpy/reactpy/backend/utils.py

```diff
@@ -8,62 +8,50 @@
 from importlib import import_module
 from typing import Any
 
-from reactpy.backend.types import BackendImplementation
+from reactpy.backend.types import BackendType
 from reactpy.types import RootComponentConstructor
 
 logger = logging.getLogger(__name__)
 
-SUPPORTED_PACKAGES = (
-    "starlette",
+SUPPORTED_BACKENDS = (
     "fastapi",
     "sanic",
     "tornado",
     "flask",
+    "starlette",
 )
 
 
 def run(
     component: RootComponentConstructor,
     host: str = "127.0.0.1",
     port: int | None = None,
-    implementation: BackendImplementation[Any] | None = None,
+    implementation: BackendType[Any] | None = None,
 ) -> None:
     """Run a component with a development server"""
     logger.warning(_DEVELOPMENT_RUN_FUNC_WARNING)
 
     implementation = implementation or import_module("reactpy.backend.default")
-
     app = implementation.create_development_app()
     implementation.configure(app, component)
-
-    host = host
     port = port or find_available_port(host)
-
     app_cls = type(app)
+
     logger.info(
-        f"Running with {app_cls.__module__}.{app_cls.__name__} at http://{host}:{port}"
+        "ReactPy is running with '%s.%s' at http://%s:%s",
+        app_cls.__module__,
+        app_cls.__name__,
+        host,
+        port,
     )
-
     asyncio.run(implementation.serve_development_app(app, host, port))
 
 
-def find_available_port(
-    host: str,
-    port_min: int = 8000,
-    port_max: int = 9000,
-    allow_reuse_waiting_ports: bool = True,
-) -> int:
+def find_available_port(host: str, port_min: int = 8000, port_max: int = 9000) -> int:
     """Get a port that's available for the given host and port range"""
     for port in range(port_min, port_max):
         with closing(socket.socket()) as sock:
             try:
-                if allow_reuse_waiting_ports:
-                    # As per this answer: https://stackoverflow.com/a/19247688/3159288
-                    # setting can be somewhat unreliable because we allow the use of
-                    # ports that are stuck in TIME_WAIT. However, not setting the option
-                    # means we're overly cautious and almost always use a different addr
-                    # even if it could have actually been used.
-                    sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
```

Comment:
Not sure why we're deleting this.

Reply:
Reason is mentioned in the PRs description.

Fixes a bug where in-use ports are being assigned on Windows. Deleting it didn't seem to impact Linux.

Reply:
Doesn't break on Linux, but every time you restart the server you'll get a different port. Perhaps we can just vary the behavior by platform.

Reply:
I should double check that, but that is why I recall introducing that.

Reply:
Okay let me know, if that's behavior on Linux I can just change the default when running win32.